libs/expo/betterangels/src/lib/screens/Client/Profile/ContactInfo.tsx

```diff
@@ -62,6 +76,81 @@ export default function ContactInfo(props: IProfileSectionProps) {
                 .map(({ label, value }) => (
                   <InfoCol key={label} label={label} value={value} />
                 ))}
+              {client?.clientProfile.phoneNumbers &&
+                client?.clientProfile.phoneNumbers.length > 0 && (
+                  <View>
+                    <View
+                      style={{
+                        alignItems: 'center',
+                        justifyContent: 'space-between',
+                        flexDirection: 'row',
+                        marginBottom: Spacings.sm,
+                      }}
+                    >
+                      <TextRegular size="sm">Phone Number(s)</TextRegular>
+                      <TextBold size="sm">Primary</TextBold>
+                    </View>
+                    {client?.clientProfile.phoneNumbers?.map((phoneNumber) => (
+                      <View
+                        style={{
+                          flexDirection: 'row',
+                          alignItems: 'center',
+                          justifyContent: 'space-between',
+                        }}
+                        key={phoneNumber.id}
+                      >
+                        <TextBold size="sm">{phoneNumber.number}</TextBold>
+                        <PawIcon />
+                      </View>
+                    ))}
+                  </View>
+                )}
+
+              <InfoCol
+                row
+                label="Email"
+                value={client?.clientProfile.user.email}
+              />
+              {client?.clientProfile.socialMediaProfiles &&
+                client?.clientProfile.socialMediaProfiles?.length > 0 && (
+                  <View>
+                    <TextRegular size="sm" mb="sm">
+                      Social Medias
+                    </TextRegular>
+                    {client?.clientProfile.socialMediaProfiles?.map(
+                      (profile) => (
+                        <View
+                          style={{
+                            flexDirection: 'row',
+                            alignItems: 'center',
+                            justifyContent: 'space-between',
+                          }}
+                          key={profile.id}
+                        >
+                          <TextRegular size="sm">
+                            {enumDisplaySocialMedia[profile.platform]}
+                          </TextRegular>
+                          <TextBold size="sm">
```

Comment:
not blocking, but maybe add a slight`margin` - these usernames can be long

Reply:
added